suite-native/qr-code/src/components/QRCode.tsx

```diff
@@ -12,7 +12,7 @@ type QRCodeProps = {
 const SCREEN_WIDTH = Dimensions.get('screen').width;
 
 const MAX_QRCODE_SIZE = 250;
-const QRCODE_PADDING = 12;
+const QRCODE_PADDING = 24;
```

Comment:
you can use `utils.spacing.large * 2` in styles below

Reply:
This const is used also outside of the style object.